### ISSUE TITLE
IDEA: serialized qr code for asn workflow

### ISSUE THREAD

**Issue by hvb81479** [pklaus/brother_ql_web#49]
Hi,
on https://github.com/tmaier/asn-qr-code-label-generator there is a great tool to print ASN's on A4 paper. Besides that it is a challenge to get the prints neatly placed on the labels it appears obvious to use the QL printer for this use case.
The p-touch software allows for all I admit but I need to start windows for this (see page 96 or so in the QL-820mwb manual); for printing stand alone an extra handheld scanner is needed and the overall setup doesn't appear to be convenient.
However, it would be great to get away from p-touch and Windows and have a OS-agnostic solution.
Would this be an interesting use case to take on?
Cheers, Hans

**Comment by FriedrichFroebel** [comment]
I do not think that out-of-the-box support for any type of label/matrix code etc. is something which should be implemented by the web interface. I would recommend you to generate the labels externally with your favorite editor/automation toolchain and pass them to the `brother_ql` CLI or `brother_ql_web`  (through a generic API) instead. My currently preferred solution is a combination of Jinja2 + LaTeX + `pdftocairo` to feed the CLI, but this surely is a matter of taste.
